Configure long line length

ColCh opened the pull request:
closes #63639

ColCh commented on src/vs/editor/common/config/commonEditorConfig.ts:
```diff
@@ -822,6 +822,11 @@ const editorConfiguration: IConfigurationNode = {
 		},
+		'editor.longLineLength': {
+			'type': 'integer',
+			'default': EDITOR_DEFAULTS.contribInfo.longLineLength,
```
is `contribInfo` a right place for this setting?

alexr00 replied:
I think it would go better as EDITOR_DEFAULTS.longLineLength. @alexandrudima, can you confirm?

ColCh replied:
Renamed option name to `maxTokenizationLineLength`

lvleihere replied:
god job!

alexr00 commented on src/vs/workbench/services/textMate/electron-browser/TMSyntax.ts:
```diff
@@ -448,13 +463,13 @@ class TMTokenization implements ITokenizationSupport {
 			if (!this._tokenizationWarningAlreadyShown) {
 				this._tokenizationWarningAlreadyShown = true;
-				this.notificationService.warn(nls.localize('too many characters', "Tokenization is skipped for lines longer than 20k characters for performance reasons."));
+				this.notificationService.warn(nls.localize('too many characters', 'Tokenization is skipped for long lines for performance reasons.'));
```
Can you add a bit about how this can be configured now? "The length of a long line can be set by configuring editor.longLineLength in settings."

ColCh commented:
Thank you for such detailed review comments!

Updated the code, please take a look again @alexandrudima @alexr00 
Build seems to pass in CI

TL;DR; changes
* removed changes from **editorOptions.ts**. Didn't knew that, thanks.
* renamed option name to `maxTokenizationLineLength` to be more specific
* added note on how to change it

alexandrudima commented:
:+1: Thank you!

ColCh commented:
Thank you too! 👍
